addons/html_editor/static/tests/link/popover.test.js

```diff
@@ -500,6 +500,17 @@ describe("Link formatting in the popover", () => {
 });
 
 describe("shortcut", () => {
+    test("create link shortcut should be at the first", async () => {
+        const { editor } = await setupEditor(`<p>[]</p>`);
+        editor.services.command.add("A test command", () => {}, {
+            hotkey: "alt+k",
+            category: "app",
+        });
+
+        press(["ctrl", "k"]);
```

Comment:
I'm a bit confused by this test because it uses alt+k then ctrl+k. Do we expect this comment to appear in the results, or not? Because in any cayse the first one is still going to be "Create link", right? So what's going on? ^^'

Reply:
It's to create another command with a lower priority in the command palette so we can check if `Create link` is placed to the first. Cause without this command, `Create link` is the only command in the palette when testing